packages/app-cli/tests/enex_to_md/checklist.md

```diff
@@ -0,0 +1,7 @@
+- [ ] One
+
+- [X] Two
+
+- [ ] Three
+
```

Comment:
Is that expected this white space between the list items?

Reply:
I thought it was expected behavior, since that's what's happening with the list5 test, so I assumed it was meant to import them with those blank lines for some reason. But now I see the other listX files don't have that blank space, so maybe list5 shouldn't have that either? 

I think it's a bug in the tests https://github.com/laurent22/joplin/issues/7348

Reply:
I don't really know. If it never came up as a bug report before I guess it's fine.